cms/djangoapps/contentstore/views/course.py

```diff
@@ -738,8 +743,17 @@ def _create_new_course(request, org, number, run, fields):
     Returns the URL for the course overview page.
     Raises DuplicateCourseError if the course already exists
     """
+    org_data = get_organization_by_short_name(org)
+    if not org_data and organizations_enabled():
+        return JsonResponse(
+            {'error': _('You must link this course to an organization in order to continue. '
+                        'Organization you selected does not exist in the system, '
+                        'you will need to add it to the system')},
```

Comment:
is the error message need to update as described in story ? 

`You must link this course to an organization in order to continue. If the organization does not appear in the list, you will need to add it to the system and refresh this page.`